tests/distributed/test_join_local_from_remote/flow.yml

```diff
@@ -20,7 +20,7 @@ pods:
     port_expose: 8000
     needs: [gateway]
   - name: join_all
-    uses: _merge
+    uses: _pass
```

Comment:
What's the motivation for changing this? I think `_merge` also works.

Reply:
To keep the usage as before. The difference being.

If _pass is used, the request of a branch is passed to the gateway as they come, if use _merge, it waits for the other part